modules/apps/analytics/analytics-client-js/test/analytics.js

```diff
@@ -290,6 +296,52 @@ describe('Analytics', () => {
 			]);
 		});
 
+		it('returns a type error if the eventId is not a string', () => {
+			const eventId = {test: 'test'};
+
+			console.error = jest.fn((val) => val);
+
+			Analytics.track(eventId);
```

Comment:
This `Analytics.track` might need to have an await oparator, once `addItem` function is returning a promise after we fixed this bug
https://github.com/brianchandotcom/liferay-portal/pull/111121

Reply:
So, this whole test case should be async

Reply:
Ok, I'll wait for your PR to be merged first.